src/lib/models/text.cpp

```diff
@@ -109,8 +109,8 @@ void Text::commitPreedit()
     // we would expect the text editor to just update the surrounding text.
     // Raises the question whether we should have commitPreedit here at all,
     // but it does preserve some consistency at least.
-    m_surrounding = m_preedit;
-    m_surrounding_offset = m_preedit.length();
+    m_surrounding += m_preedit;
```

Comment:
These changes were needed otherwise the test case "auto-correct enabled, check two spaces does insert full-stop after )" was failing: the surrounding text was emptied, so the check here was failing: https://github.com/ubports/keyboard-component/blob/master/src/view/abstracttexteditor.cpp#L534

With this change the tests pass, but I wonder if I should make sure that `m_surrounding` doesn't grow too big (should I set a limit to 256 chars, or what?).